<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" total="4" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Data Driven Testing for login Module" duration-ms="10555" started-at="2021-10-23T14:45:28Z" finished-at="2021-10-23T14:45:38Z">
    <groups>
    </groups>
    <test name="Locked out user Test" duration-ms="7419" started-at="2021-10-23T14:45:28Z" finished-at="2021-10-23T14:45:35Z">
      <class name="com.saucedemo.Test.validateLoginTestData">
        <test-method status="PASS" signature="setupTest()[pri:0, instance:com.saucedemo.Test.validateLoginTestData@5c7fa833]" name="setupTest" is-config="true" duration-ms="3565" started-at="2021-10-23T10:45:28Z" finished-at="2021-10-23T10:45:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupTest -->
        <test-method status="PASS" signature="validateLoginData(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.saucedemo.Test.validateLoginTestData@5c7fa833]" name="validateLoginData" duration-ms="2875" started-at="2021-10-23T10:45:31Z" finished-at="2021-10-23T10:45:34Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[locked_out_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[https://www.saucedemo.com/]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateLoginData -->
        <test-method status="PASS" signature="quitTest()[pri:0, instance:com.saucedemo.Test.validateLoginTestData@5c7fa833]" name="quitTest" is-config="true" duration-ms="971" started-at="2021-10-23T10:45:34Z" finished-at="2021-10-23T10:45:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitTest -->
      </class> <!-- com.saucedemo.Test.validateLoginTestData -->
    </test> <!-- Locked out user Test -->
    <test name="Problem user Test" duration-ms="7525" started-at="2021-10-23T14:45:28Z" finished-at="2021-10-23T14:45:35Z">
      <class name="com.saucedemo.Test.validateLoginTestData">
        <test-method status="PASS" signature="setupTest()[pri:0, instance:com.saucedemo.Test.validateLoginTestData@51565ec2]" name="setupTest" is-config="true" duration-ms="3642" started-at="2021-10-23T10:45:28Z" finished-at="2021-10-23T10:45:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupTest -->
        <test-method status="PASS" signature="validateLoginData(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.saucedemo.Test.validateLoginTestData@51565ec2]" name="validateLoginData" duration-ms="2948" started-at="2021-10-23T10:45:31Z" finished-at="2021-10-23T10:45:34Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[problem_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[https://www.saucedemo.com/inventory.html]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateLoginData -->
        <test-method status="PASS" signature="quitTest()[pri:0, instance:com.saucedemo.Test.validateLoginTestData@51565ec2]" name="quitTest" is-config="true" duration-ms="928" started-at="2021-10-23T10:45:34Z" finished-at="2021-10-23T10:45:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitTest -->
      </class> <!-- com.saucedemo.Test.validateLoginTestData -->
    </test> <!-- Problem user Test -->
    <test name="Performance glitch user Test" duration-ms="7868" started-at="2021-10-23T14:45:28Z" finished-at="2021-10-23T14:45:36Z">
      <class name="com.saucedemo.Test.validateLoginTestData">
        <test-method status="PASS" signature="setupTest()[pri:0, instance:com.saucedemo.Test.validateLoginTestData@215be6bb]" name="setupTest" is-config="true" duration-ms="3543" started-at="2021-10-23T10:45:28Z" finished-at="2021-10-23T10:45:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupTest -->
        <test-method status="FAIL" signature="validateLoginData(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.saucedemo.Test.validateLoginTestData@215be6bb]" name="validateLoginData" duration-ms="3532" started-at="2021-10-23T10:45:31Z" finished-at="2021-10-23T10:45:35Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[performance_glitch_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[https://www.saucedemo.com/inventory.html]]>
              </value>
            </param>
          </params>
          <exception class="org.openqa.selenium.NoSuchWindowException">
            <message>
              <![CDATA[no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=94.0.4606.81)
  (Driver info: chromedriver=94.0.4606.61 (418b78f5838ed0b1c69bb4e51ea0252171854915-refs/branch-heads/4606@{#1204}),platform=Windows NT 10.0.19042 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 11 milliseconds
Build info: version: '2.53.0', revision: '35ae25b', time: '2016-03-15 16:57:40'
System info: host: 'DESKTOP-E69K95O', ip: '192.168.5.67', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_301'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{mobileEmulationEnabled=false, timeouts={implicit=0, pageLoad=300000, script=30000}, hasTouchScreen=false, platform=XP, acceptSslCerts=false, goog:chromeOptions={debuggerAddress=localhost:3903}, webauthn:extension:credBlob=true, acceptInsecureCerts=false, webStorageEnabled=true, browserName=chrome, takesScreenshot=true, javascriptEnabled=true, setWindowRect=true, webauthn:extension:largeBlob=true, unexpectedAlertBehaviour=ignore, applicationCacheEnabled=false, rotatable=false, networkConnectionEnabled=false, chrome={chromedriverVersion=94.0.4606.61 (418b78f5838ed0b1c69bb4e51ea0252171854915-refs/branch-heads/4606@{#1204}), userDataDir=C:\Users\AGILE1~1\AppData\Local\Temp\scoped_dir4072_1954750214}, takesHeapSnapshot=true, pageLoadStrategy=normal, strictFileInteractability=false, databaseEnabled=false, handlesAlerts=true, version=94.0.4606.81, browserConnectionEnabled=false, proxy={}, nativeEvents=true, locationContextEnabled=true, cssSelectorsEnabled=true, webauthn:virtualAuthenticators=true}]
Session ID: b2cbbbbf4c013ec23b139197c259476c]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchWindowException: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=94.0.4606.81)
  (Driver info: chromedriver=94.0.4606.61 (418b78f5838ed0b1c69bb4e51ea0252171854915-refs/branch-heads/4606@{#1204}),platform=Windows NT 10.0.19042 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 11 milliseconds
Build info: version: '2.53.0', revision: '35ae25b', time: '2016-03-15 16:57:40'
System info: host: 'DESKTOP-E69K95O', ip: '192.168.5.67', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_301'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{mobileEmulationEnabled=false, timeouts={implicit=0, pageLoad=300000, script=30000}, hasTouchScreen=false, platform=XP, acceptSslCerts=false, goog:chromeOptions={debuggerAddress=localhost:3903}, webauthn:extension:credBlob=true, acceptInsecureCerts=false, webStorageEnabled=true, browserName=chrome, takesScreenshot=true, javascriptEnabled=true, setWindowRect=true, webauthn:extension:largeBlob=true, unexpectedAlertBehaviour=ignore, applicationCacheEnabled=false, rotatable=false, networkConnectionEnabled=false, chrome={chromedriverVersion=94.0.4606.61 (418b78f5838ed0b1c69bb4e51ea0252171854915-refs/branch-heads/4606@{#1204}), userDataDir=C:\Users\AGILE1~1\AppData\Local\Temp\scoped_dir4072_1954750214}, takesHeapSnapshot=true, pageLoadStrategy=normal, strictFileInteractability=false, databaseEnabled=false, handlesAlerts=true, version=94.0.4606.81, browserConnectionEnabled=false, proxy={}, nativeEvents=true, locationContextEnabled=true, cssSelectorsEnabled=true, webauthn:virtualAuthenticators=true}]
Session ID: b2cbbbbf4c013ec23b139197c259476c
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:158)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:678)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:701)
	at org.openqa.selenium.remote.RemoteWebDriver.getCurrentUrl(RemoteWebDriver.java:326)
	at com.saucedemo.Test.validateLoginTestData.validateLoginData(validateLoginTestData.java:53)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:393)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchWindowException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateLoginData -->
        <test-method status="PASS" signature="quitTest()[pri:0, instance:com.saucedemo.Test.validateLoginTestData@215be6bb]" name="quitTest" is-config="true" duration-ms="777" started-at="2021-10-23T10:45:35Z" finished-at="2021-10-23T10:45:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitTest -->
      </class> <!-- com.saucedemo.Test.validateLoginTestData -->
    </test> <!-- Performance glitch user Test -->
    <test name="Standard user Test" duration-ms="10555" started-at="2021-10-23T14:45:28Z" finished-at="2021-10-23T14:45:38Z">
      <class name="com.saucedemo.Test.validateLoginTestData">
        <test-method status="PASS" signature="setupTest()[pri:0, instance:com.saucedemo.Test.validateLoginTestData@7c16905e]" name="setupTest" is-config="true" duration-ms="3325" started-at="2021-10-23T10:45:28Z" finished-at="2021-10-23T10:45:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupTest -->
        <test-method status="FAIL" signature="validateLoginData(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.saucedemo.Test.validateLoginTestData@7c16905e]" name="validateLoginData" duration-ms="6503" started-at="2021-10-23T10:45:31Z" finished-at="2021-10-23T10:45:38Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[standard_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[https://www.saucedemo.com/inventory.html]]>
              </value>
            </param>
          </params>
          <exception class="org.openqa.selenium.WebDriverException">
            <message>
              <![CDATA[chrome not reachable
  (Session info: chrome=94.0.4606.81)
  (Driver info: chromedriver=94.0.4606.61 (418b78f5838ed0b1c69bb4e51ea0252171854915-refs/branch-heads/4606@{#1204}),platform=Windows NT 10.0.19042 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 4.16 seconds
Build info: version: '2.53.0', revision: '35ae25b', time: '2016-03-15 16:57:40'
System info: host: 'DESKTOP-E69K95O', ip: '192.168.5.67', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_301'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{mobileEmulationEnabled=false, timeouts={implicit=0, pageLoad=300000, script=30000}, hasTouchScreen=false, platform=XP, acceptSslCerts=false, goog:chromeOptions={debuggerAddress=localhost:3904}, webauthn:extension:credBlob=true, acceptInsecureCerts=false, webStorageEnabled=true, browserName=chrome, takesScreenshot=true, javascriptEnabled=true, setWindowRect=true, webauthn:extension:largeBlob=true, unexpectedAlertBehaviour=ignore, applicationCacheEnabled=false, rotatable=false, networkConnectionEnabled=false, chrome={chromedriverVersion=94.0.4606.61 (418b78f5838ed0b1c69bb4e51ea0252171854915-refs/branch-heads/4606@{#1204}), userDataDir=C:\Users\AGILE1~1\AppData\Local\Temp\scoped_dir5068_146660480}, takesHeapSnapshot=true, pageLoadStrategy=normal, strictFileInteractability=false, databaseEnabled=false, handlesAlerts=true, version=94.0.4606.81, browserConnectionEnabled=false, proxy={}, nativeEvents=true, locationContextEnabled=true, cssSelectorsEnabled=true, webauthn:virtualAuthenticators=true}]
Session ID: be9c4d00dcf74e2b9cdb180839b2aee2]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.WebDriverException: chrome not reachable
  (Session info: chrome=94.0.4606.81)
  (Driver info: chromedriver=94.0.4606.61 (418b78f5838ed0b1c69bb4e51ea0252171854915-refs/branch-heads/4606@{#1204}),platform=Windows NT 10.0.19042 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 4.16 seconds
Build info: version: '2.53.0', revision: '35ae25b', time: '2016-03-15 16:57:40'
System info: host: 'DESKTOP-E69K95O', ip: '192.168.5.67', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_301'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{mobileEmulationEnabled=false, timeouts={implicit=0, pageLoad=300000, script=30000}, hasTouchScreen=false, platform=XP, acceptSslCerts=false, goog:chromeOptions={debuggerAddress=localhost:3904}, webauthn:extension:credBlob=true, acceptInsecureCerts=false, webStorageEnabled=true, browserName=chrome, takesScreenshot=true, javascriptEnabled=true, setWindowRect=true, webauthn:extension:largeBlob=true, unexpectedAlertBehaviour=ignore, applicationCacheEnabled=false, rotatable=false, networkConnectionEnabled=false, chrome={chromedriverVersion=94.0.4606.61 (418b78f5838ed0b1c69bb4e51ea0252171854915-refs/branch-heads/4606@{#1204}), userDataDir=C:\Users\AGILE1~1\AppData\Local\Temp\scoped_dir5068_146660480}, takesHeapSnapshot=true, pageLoadStrategy=normal, strictFileInteractability=false, databaseEnabled=false, handlesAlerts=true, version=94.0.4606.81, browserConnectionEnabled=false, proxy={}, nativeEvents=true, locationContextEnabled=true, cssSelectorsEnabled=true, webauthn:virtualAuthenticators=true}]
Session ID: be9c4d00dcf74e2b9cdb180839b2aee2
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:158)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:678)
	at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:327)
	at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:85)
	at com.saucedemo.Test.validateLoginTestData.validateLoginData(validateLoginTestData.java:49)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:393)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.WebDriverException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateLoginData -->
        <test-method status="PASS" signature="quitTest()[pri:0, instance:com.saucedemo.Test.validateLoginTestData@7c16905e]" name="quitTest" is-config="true" duration-ms="707" started-at="2021-10-23T10:45:38Z" finished-at="2021-10-23T10:45:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitTest -->
      </class> <!-- com.saucedemo.Test.validateLoginTestData -->
    </test> <!-- Standard user Test -->
  </suite> <!-- Data Driven Testing for login Module -->
</testng-results>
